test/session_test.rb

```diff
@@ -373,7 +375,7 @@ def setup
   end
 
   test "raise error if timestamp is too old" do
-    params = { code: "any-code", timestamp: Time.now - 2.days }
+    params = { code: "any-code", timestamp: Time.now - 2 * SECONDS_IN_A_DAY }
```

Comment:
Extremely nitpicky, but would it make sense to use `Session.SECONDS_IN_A_DAY` here?

Reply:
There's a couple of reasons why I usually avoid referencing a implementation's constants in the tests:

- If the constant is wrong in the implementation, and the test references that, then you lose out on the 'double-entry bookkeeping' safeguard of writing a test.
- If I choose to rename the constant, or move it somewhere else, the tests shouldn't break since the implementation hasn't changed.